src/components/Message/Message.tsx

```diff
@@ -0,0 +1,96 @@
+import { Heading } from '@components/Heading'
+import Image from '@components/Image'
+import { Spacer } from '@components/Spacer'
+import { useStyles } from '@helpers/hooks/useStyles'
+import { FC, Fragment, ReactNode } from 'react'
+
+import styles from './Message.module.scss'
+
+interface MessageProps {
+  children?: ReactNode
+  className?: string | string[]
+  title?: string
+  message?: string
+  url?: string
+  height?: number
+  titleLevel?: 1 | 2 | 3 | 4 | 5 | 6 | 'large'
+  messageLevel?: 1 | 2 | 3 | 4 | 5 | 6 | 'large'
+  video?: ReactNode
+  multimediaPosition?: 'before' | 'after'
+  bold?: boolean
+  withoutMargin?: boolean
+}
+
+export const Message: FC<MessageProps> = ({
+  children = null,
+  className = '',
+  title = undefined,
+  message = undefined,
+  url = undefined,
```

Comment:
Moim zdaniem nie potrzeba definiować w default czegoś co jest undefined, jak się nie przekaże tego propsa no to domyślnie taki on będzie, jak wyżej jest optional